composer/algorithms/colout/colout.py

```diff
@@ -39,16 +69,25 @@ def colout_batch(input: ImgT, p_row: float = 0.15, p_col: float = 0.15) -> ImgT:
             new_X = colout_batch(X_example, p_row=0.15, p_col=0.15)
 
     Args:
-        input: :class:`PIL.Image.Image` or :class:`torch.Tensor` of image data. In
-            the latter case, must be a single image of shape ``CHW`` or a batch
-            of images of shape ``NCHW``.
-        p_row: Fraction of rows to drop (drop along H). Default: ``0.15``.
-        p_col: Fraction of columns to drop (drop along W). Default: ``0.15``.
+        sample (torch.Tensor | PIL.Image | Tuple[torch.Tensor, torch.Tensor] | Tuple[PIL.Image, PIL.Image]:):
+            Either a single tensor or image or a 2-tuple of tensors or images. When tensor(s), the tensor must be of shape
+            ``CHW`` for a single image or ``NCHW`` for a batch of images of shape.
+        p_row (float, optional): Fraction of rows to drop (drop along H). Default: ``0.15``.
+        p_col (float, optional): Fraction of columns to drop (drop along W). Default: ``0.15``.
+        resize_target (bool | str, optional): If ``sample`` is a tuple, whether to resize both objects in the tuple.
+            If set to 'auto', resizing both objects will be based on if the objects hav the same spatial dimensions.
```

Comment:
Nit: "hav" -> "have"

Or maybe, "If set to 'auto', both objects will be resized if they have the same spatial dimensions."

Reply:
Ooops. Done!